content/hardware/02.hero/boards/uno-r4-minima/tutorials/dac/dac.md

```diff
@@ -38,7 +38,7 @@ The circuit required for this tutorial can be found in the diagram below:
 
 ![Piezo buzzer connected to UNO R4](./assets/circuit.png)
 
-## Analog Output VS. PWM
+## Analog Output vs PWM
 For many use cases when analog output is required, using PWM (Pulse Width Modulation) instead of genuine analog output will yield essentially the same results. A digital output pin can only either be fully on (HIGH) or fully off (LOW), but by turning on and off very quickly with precise timings, the average voltage can be controlled and emulate an analog output. This method is called [PWM](/learn/microcontrollers/analog-output). 
```

Comment:
```suggestion
For many use cases when analog output is required, using PWM (Pulse Width Modulation) instead of genuine analog output will yield essentially the same results. A digital output pin can only either be fully on (HIGH) or fully off (LOW), but by turning it on and off very quickly with precise timings, the average voltage can be controlled and emulate an analog output. This method is called [PWM](/learn/microcontrollers/analog-output). 
```